selfdrive/debug/README.md

```diff
@@ -34,3 +34,27 @@ optional arguments:
   --addr ADDR
   --values VALUES  values to monitor (instead of entire event)
 ```
+
+## [vw_mqb_config.py](vw_mqb_config.py)
+
+Shows Volkswagen EPS software and coding info, and enables or disables Heading
```

Comment:
think this should go in the argparser description

Reply:
Are you proposing to move the entirety of these two paragraphs into the tool, or just this first sentence?

Reply:
Went ahead and moved all of it, let me know if that wasn't what you had in mind.